src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/delete/actions/DeallocateResourcesAction.java

```diff
@@ -48,6 +48,7 @@ public void perform(State from, State to, Event event, FlowDeleteContext context
             resourcesManager.deallocateMirrorGroup(mirrorPathResources.getFlowPathId(),
                     mirrorPathResources.getMirrorSwitchId());
         });
+        // TODO deallocate mirrors
```

Comment:
Maybe to add a link to the issue? In my experience, when there is no issue, people tend to forget about TODOs